showcase/build/iaf_cond_alpha_implicit.h

```diff
@@ -0,0 +1,888 @@
+  *       - If State_ contained @c Time members, you need to define the
+  *         assignment operator to recalibrate all members of type @c Time . You
+  *         may also want to define the assignment operator.
+  *       - If State_ contained members that cannot copy themselves, such
+  *         as C-style arrays, you need to define the copy constructor and
+  *         assignment operator to copy those members.
+  */
+  struct State_ {
+    //! Symbolic indices to the elements of the state vector y
+    enum StateVecElems { V_m, g_in, g_ex, __D_g_in, __D_g_ex, STATE_VEC_SIZE };
+    //! state vector, must be C-array for GSL solver
+    double y[STATE_VEC_SIZE];
+
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
+
+    double I_syn_exc; //
+
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
+
+    double I_syn_inh; //
+
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
+
+    double I_leak; //
+
+    State_();
+  };
+
+  /**
+  * Internal variables of the neuron.
+  *
+  *
+  *
+  * These variables must be initialized by @c calibrate, which is called before
+  * the first call to @c update() upon each call to @c Simulate.
+  * @node Variables_ needs neither constructor, copy constructor or assignment
+  * operator,
+  *       since it is initialized by @c calibrate(). If Variables_ has members
+  * that
+  *       cannot destroy themselves, Variables_ will need a destructor.
+  */
+  struct Variables_ {
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
+
+    double PSConInit_E; //
+
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
+
+    double PSConInit_I; //
+
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
+
+    long RefractoryCounts; //
+
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
+
+    long r; //
+  };
+
+  /**
+    * Buffers of the neuron.
+    * Ususally buffers for incoming spikes and data logged for analog recorders.
+    * Buffers must be initialized by @c init_buffers_(), which is called before
+    * @c calibrate() on the first call to @c Simulate after the start of NEST,
+    * ResetKernel or ResetNetwork.
+    * @node Buffers_ needs neither constructor, copy constructor or assignment
+   * operator,
+    *       since it is initialized by @c init_nodes_(). If Buffers_ has members
+   * that
+    *       cannot destroy themselves, Buffers_ will need a destructor.
+    */
+  struct Buffers_ {
+    Buffers_(iaf_cond_alpha_implicit &);
+    Buffers_(const Buffers_ &, iaf_cond_alpha_implicit &);
+
+    /** Logger for all analog data */
+    nest::UniversalDataLogger<iaf_cond_alpha_implicit> logger_;
+
+    inline nest::RingBuffer &get_spikeInh() { return spikeInh; }
+    nest::RingBuffer spikeInh;
+    //!< Buffer incoming Buffers through delay, as sum
+    ;
+    double spikeInh_last_value_;
```

Comment:
what is the role of this variable?
Looking at the [update function](https://github.com/nest/nestml/pull/340/files#diff-637b71d8b3c68e70a878b55451944644R406), I don't think these ``last`` variables are necessary...